lldb/include/lldb/API/SBValue.h

```diff
@@ -374,6 +374,34 @@ class LLDB_API SBValue {
   lldb::SBWatchpoint WatchPointee(bool resolve_location, bool read, bool write,
                                   SBError &error);
 
+  /// If this value represents a C++ class that has a vtable, return an value
+  /// that represents the virtual function table.
+  ///
+  /// SBValue::GetError() will be in the success state if this value represents
+  /// a C++ class with a vtable, or an appropriate error describing that the
+  /// object isn't a C++ class with a vtable or not a C++ class.
+  ///
+  /// SBValue::GetName() will be the demangled symbol name for the virtual
+  /// function table like "vtable for Baseclass".
```

Comment:
Shouldn't this be `vtable for ChildClass` in RTTI domain?

Reply:
I think "BaseClass" was just an example here, but maybe a more abstract name would be better.  

For an actual C++ object, the object's vtable pointer always points to the most specific class of the object, a fact that lldb relies on to do C++ dynamic type detection.

Reply:
I can change it to "vtable for <classname>"?

Reply:
Sounds good